numpyro/infer/elbo.py

```diff
@@ -34,9 +34,9 @@ class ELBO:
 
     :param num_particles: The number of particles/samples used to form the ELBO
         (gradient) estimators.
-    :param vectorize_particles: Whether to use `jax.vmap` to compute ELBOs over the
-        num_particles-many particles in parallel. If False use `jax.lax.map`.
-        Defaults to True.
+    :param vectorize_particles: Callable to vectorize the computation of ELBOs over
```

Comment:
this change breaks current users. You can allow callable and check callable first. Defaults still to True (vmap).